project/legislation/legislatorslocation.py

```diff
@@ -0,0 +1,35 @@
+import Model from model
+
+def get_legislator_and_district(account):
```

Comment:
For simplicity, lets only ever retrieve the results directly from our database.

The database will be synchronized in a separate process.

In effect, we are trying to achieve separation of concerns here, where one concern is retrieving the data from a local 'mirror' and the other concern is synchronizing that 'mirror' via a scheduled task.

Reply:
In terms of the model, consider making separate tables for 'legislator' and 'district' with a relationship between the two. That way, the district data are only defined in one place, rather than duplicated in each representative.